src/utils/consts/timeConsts.ts

```diff
@@ -1,2 +1,2 @@
-export const ONESECOND = 1000;
-export const FIVESECONDS = 5000;
+export const ONE_SECOND = 1000;
+export const FIVE_SECONDS = 5000;
```

Comment:
its not informative. it should represent what does it mean. - DEBOUNCE_DELAY or AMOUNT_INCREASE, you just change number with word